molecule/provisioner/lint/ansible_lint.py

```diff
@@ -54,6 +54,28 @@ class AnsibleLint(base.Base):
 
     The `x` option has to be passed like this due to a `bug`_ in Ansible Lint.
 
+    An additional option is to use the following option for ansible-lint on `molecule.yml`
```

Comment:
We should use ` ``molecule.yml`` ` here, since Sphinx formatting requires double quotes for these code blocks.